src/services/facilityService.ts

```diff
@@ -0,0 +1,806 @@
+import * as gqlTypes from '../typeDefs/gqlTypes.js'
+import { db } from '../kyselyClient.js'
+import type { FacilitiesTable } from '../typeDefs/kyselyTypes.js'
+import type { Selectable } from 'kysely'
+import { ErrorCode, Result } from '../result.js'
+import { logger } from '../logger.js'
+import { getSupabaseClient } from '../supabaseClient.js'
+import { createAuditLog } from './auditLogServiceSupabase.js'
+import { validateIdInput, validateCreateFacilityInput, validateFacilitiesSearchInput, validateUpdateFacilityInput } from '../validation/validateFacility.js'
+
+// Capabilities for Supabase-like query builders
+export type HasIlike = {
```

Comment:
This is a little confusing, is this just for the database compatibility? If so, maybe we can put in the database file instead